datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -7965,6 +7965,10 @@ type ActorFilter {
   """
   resourceOwners: Boolean!
 
+  resourceOwnersTypesUrns: [String!]
```

Comment:
Please add comments at parity with the other fields!

Reply:
Let's also namethis "resourceOwnerTypes" 

I think given the new OwnershipTypes feature we will assume this is an URN. You can also place this in the field comment

Reply:
Added comments and changed name to `resourceOwnersTypes`